FSJ_django20_project/FSJ/urls.py

```diff
@@ -29,9 +29,9 @@
     path('coord_awardslist/add/', views.coordinator_add_awards, name = 'coord_addaward'),
     path('coord_awardslist/action/', views.coordinator_awardaction, name='coord_awardaction'),
     path('coord_awardslist/<str:award_idnum>/', views.coordinator_awardedit, name = 'coord_awardedit'),
-    path('programs/list_programs/', views.list_programs, name='list_programs'),
+    path('programs/', views.list_programs, name='list_programs'),
     path('programs/add/', views.add_program, name='add_program'),
-    path('programs/edit/<str:program_code>/', views.edit_program, name='edit_program'),
+    path('programs/<str:program_code>/', views.edit_program, name='edit_program'),
```

Comment:
This change makes programs fall into the same bug as years of study, students, adjudicators, etc. wherein if the pk = "add" or "delete" then the system gets messed up. Additionally, those defects are supposed to be fixed soon so without knowing if the fix will be done for each case to ALWAYS allow the case that "from a model page clicking cancel we can always navigate to href="../" without concern seems problematic. What is the reason we didn't add a return_url as with other places?